src/redux/restReducers/kontaktinformasjon.ts

```diff
@@ -1,17 +1,14 @@
-import { createActionsAndReducerDeprecated } from './deprecatedRestResource';
-import { getKontaktinformasjon } from '../../api/kontaktinformasjon-api';
-import { KRRKontaktinformasjon } from '../../models/kontaktinformasjon';
+import { createRestResourceReducer } from '../../rest/utils/restResource';
+import { apiBaseUri } from '../../api/config';
+import { AppState } from '../reducers';
 
-const { reducer, action, actionNames } = createActionsAndReducerDeprecated('kontaktinformasjon');
-
-export function hentKontaktinformasjon(fødselsnummer: string) {
-    return action(() =>
-        getKontaktinformasjon(fødselsnummer).then((data: KRRKontaktinformasjon) => {
-            return data;
-        })
-    );
+export function getKontaktinformasjonFetchUri(state: AppState) {
+    const fnr = state.gjeldendeBruker.fødselsnummer;
+    return `${apiBaseUri}/person/${fnr}/kontaktinformasjon`;
 }
 
+const { reducer, actionNames } = createRestResourceReducer('kontaktinformasjon', getKontaktinformasjonFetchUri);
```

Comment:
Burde ikke dette navnet si noe om at den lager masse andre nyttige funksjoner? Likte "and actions" fra sist navn.

Reply:
Aha, enig 👍